workout.py

```diff
@@ -100,7 +100,7 @@ def run_workout(workout, debug=False):
         if not debug:
             run_exercise(e.exercise)
         else:
-            logging.info("QA MODE: Running exercise: %s" % e)
+            logging.info("QA MODE: Running exercise: %s" % e.exercise)
```

Comment:
You could just log via `logging.debug()` and change the change the logging level when running with `-d`

Reply:
That's a great idea. I think I'm going to do that in a separate PR since I'll probably want to use a separate logger instead of the default one at that point